src/cork.rs

```diff
@@ -48,10 +58,12 @@ impl proto::contract_call_service_server::ContractCallService for CorkHandler {
         }
 
         debug!("checking publisher authorization for {cellar_id}");
-        handle_authorization(&cellar_id, certs).await?;
+        let subscription_id = format!("{}:{}", chain_id, cellar_id.trim().to_lowercase());
```

Comment:
These aren't normalized to lowercase, the subscription IDs reflect whatever the output of querying the cellar IDs from the chain is, which after checking is the mixed-case checksum version. If we aren't, we may want to consider reading in the cellar address inputs into Eth types and stringifying them to ensure we get that value.